routers/api/v1/repo/file.go

```diff
@@ -33,6 +33,10 @@ import (
 	files_service "code.gitea.io/gitea/services/repository/files"
 )
 
+const (
+	giteaObjectTypeHeader = "X-Gitea-Object-Type"
+)
```

Comment:
```suggestion
const giteaObjectTypeHeader = "X-Gitea-Object-Type"
```

Reply:
if you like to add another const ... and gofumpt does not complain either

Reply:
Yup, but here is only one at the moment .....